tests/sles4sap/hana_install.pm

```diff
@@ -211,7 +211,7 @@ sub run {
     # hdblcm is used for installation, verify if it exists
     my $hdblcm = '/sapinst/' . get_var('HANA_HDBLCM', "DATA_UNITS/HDB_SERVER_LINUX_" . uc(get_required_var('ARCH')) . "/hdblcm");
     die "hdblcm is not in [$hdblcm]. Set HANA_HDBLCM to the appropiate relative path. Example: DATA_UNITS/HDB_SERVER_LINUX_X86_64/hdblcm"
-      if (script_run "ls $hdblcm");
+      if (script_run "ls $hdblcm /sapinst/hdblcm");
```

Comment:
Don't hard-code the path here. Set `HANA_HDBLCM: 'hdbclm'` in the test settings instead, or change the default in L212.